bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml

```diff
@@ -571,7 +571,6 @@ spec:
                       - containerPort: 9443
                         name: https
                     resources: {}
-                    securityContext: {}
```

Comment:
Is `securityContext` not needed ?

Reply:
nope, we remove it to allow Openshift set the SecurityContext that works better for the [securityContextConstraints](https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/authorization.html#security-context-constraints) of the Openshift Project (AKA Namespace)

Reply:
ok